lib/jekyll-import/importers/ghost.rb

```diff
@@ -26,7 +28,6 @@ def self.process(options)
         end
       end
 
-      private
```

Comment:
This can be emulated with `private_class_method` and is something I think we should consider using here.

Reply:
Rubocop is still not happy: `W: Lint/UselessAccessModifier: Useless private_class_method access modifier.`